godtossing/Pos/Ssc/GodTossing/LocalData/Logic.hs

```diff
@@ -254,7 +253,7 @@ sscProcessDataDo richmenData bvd gs payload =
         let multiRichmen = HM.fromList [richmenData]
         unless (storedEpoch == givenEpoch) $
             throwError $ DifferentEpoches storedEpoch givenEpoch
-        let maxMemPoolSize = bvdMaxBlockSize bvd * memPoolLimitRatio
+        let maxMemPoolSize = bvdMaxBlockSize bvd
```

Comment:
I don't think this is right, as far as I can see these things have different units. I think the `bvdMaxBlockSize` is the maximum allowed size of a block in **bytes** not in the number of transactions.

Also, I thought we were getting the pool size from the constants, not setting it based on how many transactions can fit in a block.